packages/legacy-workbench/src/js/angular/guides/tour-lib-services/shepherd.service.js

```diff
@@ -921,11 +921,18 @@ function ShepherdService($location, $translate, LocalStorageAdapter, LSKeys, $ro
 
         // We have some Angular bits on the elements and this is needed to make them work
         const scope = angular.element(currentStepElement).scope();
-        if (angular.isFunction(stepDescription.onScope)) {
+        // After the migration, the scope here is always undefined.
+        // This broke the "copy query to editor" functionality in the execute-sparql-query plugin.
+        // Note: The scope is used only in the execute-sparql-query plugin.
+        // TODO: To restore this functionality, it needs to be reimplemented in a way that does not depend on AngularJS.
+        if (scope) {
+            console.log("%c Guide step scope: ", "color: green", scope);
```

Comment:
I guess that this logger is left intentionally. Also the formatting below is wrong.

Reply:
Fixed